src/main/java/com/minelittlepony/hdskins/skins/api/SkinServer.java

```diff
@@ -1,8 +1,12 @@
-package com.minelittlepony.hdskins.skins;
+package com.minelittlepony.hdskins.skins.api;
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.minelittlepony.hdskins.client.HDSkins;
+import com.minelittlepony.hdskins.skins.Feature;
+import com.minelittlepony.hdskins.skins.SkinType;
+import com.minelittlepony.hdskins.skins.SkinUpload;
```

Comment:
`SkinUpload` is a parameter to the servers. I wouldn't put it into the `skins` package.